schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromSPARQL-DCAT.xsl

```diff
@@ -402,44 +484,162 @@
                         </skos:Concept>
                     </dcat:theme>
                   -->
+                  <xsl:variable name="dcatThemes"
```

Comment:
In https://data.jrc.ec.europa.eu/dataset/98336641-fd1c-4992-8c7b-c470dd5eb81e.rdf there is a  `dcat:theme` element, but doesn't seem added to the ISO document:

```
    <dcat:theme rdf:resource="http://publications.europa.eu/resource/authority/data-theme/SOCI" />
```

Reply:
Currently the mapping expect a `prefLabel` to be set and in this example, it is not. We could add at some point more entity resolution from the RDF document. Currently a number of URI are not resolved and could miss label. eg.
* common EU vocabulary http://publications.europa.eu/resource/authority/data-theme/SOCI, http://publications.europa.eu/resource/authority/file-type/TIFF
* custom access constraints vocabulary http://data.jrc.ec.europa.eu/access-rights/no-limitations